R/checks.R

```diff
@@ -1,101 +1,114 @@
-check_numeric <- function(x, input = "`x`") {
+check_numeric <- function(x, input = "`x`", call = rlang::caller_env()) {
   if (!is.vector(x) || !is.numeric(x)) {
-    rlang::abort(paste0(input, " should be a numeric vector."))
+    cli::cli_abort("{.arg {input}} should be {an} numeric vector.")
```

Comment:
This substitution will raise an error, as there's no `an` in the fn environment.